js/controller.js

```diff
@@ -364,6 +366,16 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.renderSkin();
     },
 
+    // NYM – enable closed captions when volume is off, or disable closed captions when volume is on
+    // for v4 players that default to displaying closed captions if available
+    onVolumeChangedToggleClosedCaptions: function (event, newVolue) {
```

Comment:
should this by name-spaced with nym, i.e. `nymOnVolumeChangedToggleClosedCaptions`?